main/main.cpp

```diff
@@ -830,15 +832,16 @@ void Main::test_cleanup() {
 	if (globals) {
 		memdelete(globals);
 	}
-	if (engine) {
-		memdelete(engine);
-	}
 
 	unregister_core_driver_types();
 	unregister_core_extensions();
 	uninitialize_modules(MODULE_INITIALIZATION_LEVEL_CORE);
 	unregister_core_types();
 
+	if (engine) {
+		memdelete(engine);
+	}
```

Comment:
What's this change for?

Reply:
Huuum, I had to do this as it seemed a missed case from https://github.com/godotengine/godot/pull/92733, and without this change, a segfault would now occur with the test build, because the Engine singleton is being used during unitialization code: https://github.com/godotengine/godot/pull/92563/files#diff-dc18e6ed2e71c172f95fc57c69a508fc3a3c28bb324aa7523f10d70d71f4227bR144-R148

But now that I've checked it, the PR does this exact change already..? It was merged 2 days ago, so I might just need a rebase.

Reply:
Yup, it went away with the rebase :+1: